dts/arm/st/f4/stm32f4.dtsi

```diff
@@ -470,6 +470,17 @@
 			status = "disabled";
 			label = "DMA_2";
 		};
+
+		quadspi: quadspi@a0001000 {
+			compatible = "st,stm32-qspi";
+			#address-cells = <0x1>;
+			#size-cells = <0x0>;
+			reg = <0xa0001000 0x34>;
```

Comment:
Nitpick: Length is not used today, but for now convention is to set it as 0x400 for all devices.